.gitignore

```diff
@@ -73,3 +73,4 @@ __pycache__/
 cache/
 output/
 .DS_Store
+/examples/kelmarsh_kaggle_data
```

Comment:
something is going wrong.. the folder is ignored but the era dataset has been added to the repo.